src/main/java/picard/analysis/TheoreticalSensitivityMetrics.java

```diff
@@ -0,0 +1,14 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricBase;
+
+/**
+ * Created by fleharty on 6/30/17.
+ */
+public class TheoreticalSensitivityMetrics extends MetricBase {
+    /** Theoretical Het Sensitivity for log-odss of 6.2 **/
```

Comment:
Also, I was thinking about saving the 20/20/20 metric in this file.  It is a type of TheoreticalSensitivity, so it seems reasonable to me to put it there.  What do you think?